env.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/bash
 
-export FPGA_TOOL_PERF_DIR=`pwd`
+. ./env/conda/bin/activate
+source utils/environment.python.sh
 
 if [ -z "${VIVADO_SETTINGS}" ]; then
-    echo "ERROR: \$VIVADO_SETTINGS environmental variable needs to be set"
-    return 1
+    echo "WARNING: using default vivado settings"
+    VIVADO_SETTINGS=/opt/Xilinx/Vivado/2017.2/settings64.sh
```

Comment:
Can you add a `FIXME:` to use the conda `xilinx-vivado` virtual package when @PiotrZierhoffer's team creates that.